### PR TITLE
Replace ZTR broken link with new Weidmuller video

### LINKED ISSUE #2399: ZTR Weidmuller video private
The Weidmuller connector video at https://docs.wpilib.org/en/stable/docs/zero-to-robot/step-1/intro-to-frc-robot-wiring.html#weidmuller-connectors is no longer available and needs to be replaced.

### PR DESCRIPTION
**Pull request by jasondaming**
Fixes #2399 

This link isn't the exact same but I think this one has good info and is the best replacement.

### CODE REVIEW

**Comment by sciencewhiz** [comment]
It might be nice to link ~15 seconds in after some of the intro that teams might not care about.
Maybe we should add some reference that the connectors can be used with or without ferrules, since the video leads with the ferrule installation.

File: source/docs/zero-to-robot/step-1/intro-to-frc-robot-wiring.rst [hunk 1]
```diff
@@ -278,14 +278,15 @@ Using the Dual Lock or hardware, fasten all components to the board. Note that i
...
-        <iframe src="https://www.youtube-nocookie.com/embed/Zsnro-9_ZfQ" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
+        <iframe src="https://www.youtube-nocookie.com/embed/SfkmyN9mYLA?si=dMHW1JCVkbpDjbu6&amp;start=39" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
     </div>
 
 
 A number of the CAN and power connectors in the system use a Weidmuller LSF series wire-to-board connector. There are a few things to keep in mind when using this connector for best results:
 
 - Wire should be 16 AWG (1.5 :math:`mm^2`) to 24 AWG (0.25 :math:`mm^2`) (consult rules to verify required gauge for power wiring)
 - Wire ends should be stripped approximately 5/16 (~8 mm)"
+- A ferrule can be added to reinforce the connection and prevent splaying.
```
Comment (sciencewhiz) [inline source/docs/zero-to-robot/step-1/intro-to-frc-robot-wiring.rst:289]:
```suggestion
- A ferrule can be optionally added to reinforce the connection and prevent splaying, as shown in the video above
```

Comment (jasondaming) [reply source/docs/zero-to-robot/step-1/intro-to-frc-robot-wiring.rst:289]: I thought about adding optionally but isn't that redundant?  If I say something can be done it is naturally an option.

Comment (sciencewhiz) [reply source/docs/zero-to-robot/step-1/intro-to-frc-robot-wiring.rst:289]: Yes, but my worry is that someone just reading the steps will not realize that, so saying optionally makes it more obvious

**Pull request merged**